CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,85 @@
+# Contributor Covenant Code of Conduct
+
+## Our pledge
+
+In order to create an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of:
+
+* age
+* body size
+* disability
+* ethnicity
+* gender identity and expression
+* level of experience
+* education
+* socio-economic status
+* nationality
+* personal appearance
+* ethnicity
+* religion
+* sexual identity and orientation
+
```

Comment:
You should add `etc`, because right now, you can harasse someone on, for ex., its editor (Vi vs Emacs flamewar).

Reply:
One could argue this is covered by the `religion` clause ;-)

More seriously, I do think flamewars over tools should be covered. They happen often enough on some forums.

Reply:
Well, it's covered by the `etc` :wink: .